Release process job still failing on CLI JBang projects

```
Error:  Failures: 
Error:    CliProjectJBangTest.testCreateAppDefaults:47 Expected OK return code. Result:
result: {
  exitCode: {1},
  system_err: {Downloading JBang...
Installing JBang...
[jbang] Resolving dependencies...
[jbang] Loading MavenCoordinate [io.quarkus:quarkus-bom:pom:999-SNAPSHOT]
Error:  [ERROR] Unable to collect/resolve dependency tree for a resolution due to: Could not find artifact io.quarkus:quarkus-bom:pom:999-SNAPSHOT in mavencentral (https://repo1.maven.org/maven2/), caused by: Could not find artifact io.quarkus:quarkus-bom:pom:999-SNAPSHOT in mavencentral (https://repo1.maven.org/maven2/)
[jbang] Run with --verbose for more details
},
  system_out: {}
} ==> expected: <0> but was: <1>
Error:    CliProjectJBangTest.testCreateAppOverrides:77 Expected OK return code. Result:
result: {
  exitCode: {1},
  system_err: {[jbang] Resolving dependencies...
[jbang] Loading MavenCoordinate [io.quarkus:quarkus-bom:pom:999-SNAPSHOT]
Error:  [ERROR] Unable to collect/resolve dependency tree for a resolution due to: Could not find artifact io.quarkus:quarkus-bom:pom:999-SNAPSHOT in mavencentral (https://repo1.maven.org/maven2/), caused by: Could not find artifact io.quarkus:quarkus-bom:pom:999-SNAPSHOT in mavencentral (https://repo1.maven.org/maven2/)
[jbang] Run with --verbose for more details
},
  system_out: {}
} ==> expected: <0> but was: <1>
```

/cc @ebullient @glefloch (you had a look IIRC)

--- a/devtools/cli/pom.xml
+++ b/devtools/cli/pom.xml
@@ -156,6 +156,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <JBANG_REPO>${settings.localRepository}</JBANG_REPO>
+                    </environmentVariables>
+                </configuration>
+            </plugin>
         </plugins>
     </build>
 

